### ISSUE TITLE
notifications not respecting WATCHTOWER_NOTIFICATION_EMAIL_SUBJECTTAG

### ISSUE THREAD

**Issue by eddgeywing** [containrrr/watchtower#1270]
I have been running containrrr/watchtower from a docker-compose file on 3 different machines (home server, Synology NAS, Raspberry Pi) for several months without issue. In the past couple of weeks the WATCHTOWER_NOTIFICATION_EMAIL_SUBJECTTAG=ComputerName that is used for my email notifications is no longer respected on any of the 3 machines and instead the container ID is presented in the email subject line, ie "Watchtower updates on 913af4ea343". I have tried changing the smtp relay from gmail to another service with same results. As far as I can tell the issue started at the same time on all devices. I am on the latest 1.4.0 release.

I don't see anything unusual in the logs and I do receive the email notification:

time="2022-04-11T07:17:14-04:00" level=debug
time="2022-04-11T07:17:14-04:00" level=debug msg="Sleeping for a second to ensure the docker api client has been properly initialized."
time="2022-04-11T07:17:15-04:00" level=debug msg="Making sure everything is sane before starting"
time="2022-04-11T07:17:15-04:00" level=debug msg="Retrieving running, stopped and exited containers"
time="2022-04-11T07:17:15-04:00" level=debug msg="There are no additional watchtower containers"
time="2022-04-11T07:17:15-04:00" level=debug msg="Watchtower HTTP API skipped."
time="2022-04-11T07:17:15-04:00" level=info msg="Watchtower 1.4.0"
time="2022-04-11T07:17:15-04:00" level=info msg="Using notifications: smtp"
time="2022-04-11T07:17:15-04:00" level=info msg="Checking all containers (except explicitly disabled with label)"
time="2022-04-11T07:17:15-04:00" level=info msg="Scheduling first run: 2022-04-12 04:00:00 -0400 EDT"
time="2022-04-11T07:17:15-04:00" level=info msg="Note that the first check will be performed in 20 hours, 42 minutes, 44 seconds"

Has anyone else experienced this?

Thanks

**Comment by piksel** [comment]
Duplicate of #1205 (and fixed by #1219)
